OpenRA.Mods.Common/Scripting/Properties/DemolitionProperties.cs

```diff
@@ -0,0 +1,40 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using System;
+using System.Linq;
+using Eluant;
+using OpenRA.Activities;
+using OpenRA.Mods.Common.Activities;
+using OpenRA.Mods.Common.Traits;
+using OpenRA.Scripting;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Scripting
+{
+	[ScriptPropertyGroup("Demolition")]
+	public class DemolitionProperties : ScriptActorProperties, Requires<IMoveInfo>, Requires<C4DemolitionInfo>
+	{
+		public DemolitionProperties(ScriptContext context, Actor self)
+			: base(context, self) { }
+
+		[ScriptActorPropertyActivity]
+		[Desc("Move to a cell and demolish whatever is there.")]
+		public void GoDemolish(Actor target)
```

Comment:
I think it would be more consistent to call this `Demolish`, with a description "Move to and demolish an actor".